src/main/java/net/syscon/elite/security/AuthenticationFacade.java

```diff
@@ -48,12 +48,9 @@ public boolean isIdentifiedAuthentication() {
         final var auth = getAuthentication();
         return StringUtils.isNotBlank(MDC.get(PROXY_USER))
                 && Optional.ofNullable(auth).
-                filter(OAuth2Authentication.class::isInstance).
```

Comment:
why was all this replaced?

Reply:
OAuth2Authentication is a class from Spring Security Oauth and isn't available in Spring Security 5.  The only line of interest is the call to `isAuthenticated` and we still do that.